libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -98,6 +98,14 @@ const AP_Param::GroupInfo AP_MotorsMulticopter::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("THR_MIX_MAX", 14, AP_MotorsMulticopter, _thr_mix_max, AP_MOTORS_THR_MIX_MAX_DEFAULT),
 
+    // @Param: THR_WARN
+    // @DisplayName: Throttle Warn range
+    // @Description: Warn, if a motor output signal is closer than THR_WARN percent to minimum or maximum throttle in flight. Negative value disables warning.
+    // @Range: -1 10
+    // @Units: Percent
+    // @User: Advanced
+    AP_GROUPINFO("THR_WARN", 15, AP_MotorsMulticopter, _thr_warn_threshold, AP_MOTORS_THR_WARN_DEFAULT),
+
```

Comment:
@hsteinhaus 
Can I sugest to use this parameter as the interval between messages ? i.e. if zero no warnings (current behavior) and positive would be the interval between warnings, up to a predetermined nr of seconds (300s ? 600s ?).

This would effectively replace 
+#define MAX_MOTOR_EVENT_RATE